<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1392</width>
    <height>783</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1392</width>
    <height>783</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1392</width>
    <height>783</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>날짜 입력 (예: 20XX.XX.XX or 20XX-XX-XX)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>조회</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>업데이트</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>1371</width>
     <height>341</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1371</width>
     <height>341</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1371</width>
     <height>341</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTreeView" name="treeView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1351</width>
      <height>321</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1351</width>
      <height>321</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1351</width>
      <height>321</height>
     </size>
    </property>
    <attribute name="headerVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="headerStretchLastSection">
     <bool>false</bool>
    </attribute>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>1371</width>
     <height>351</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1371</width>
     <height>351</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1371</width>
     <height>351</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTreeView" name="treeView_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1351</width>
      <height>331</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1351</width>
      <height>331</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1351</width>
      <height>331</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
